tooling/helpers/evaluate_helper.py

```diff
@@ -0,0 +1,96 @@
+    get_file_id,
+    load_export_file_list
+)
+import jiwer
+import pandas
+from Levenshtein import distance as lev
+
+def load_benchmark_result_from_file(filepath: str) -> BenchmarkResult | None:
+    try:
+        with open(filepath,"r") as filehandle:
+            content = json.load(filehandle)
+        return BenchmarkResult(**content)
+    except Exception as e:
+        print(f"Could not read file {filepath} due to an error: {e}")
+        return None
+
+def eval_export_dir() -> pandas.DataFrame:
+    export_files = load_export_file_list()
+    # Axes type = list type -> dont listen to type errors if present
+    rest_df = pandas.DataFrame(columns=['file_id','duration','wer'])
+    websocket_df = pandas.DataFrame(columns=['file_id','duration','wer', 'average_levenshtein_distance'])
+    for filepath in export_files:
+        if (benchmark_res := load_benchmark_result_from_file(filepath)) is None:
+            continue
+        
+        if benchmark_res.expected_transcription is None:
+            continue
+
+        if benchmark_res.rest is not None: 
+            if benchmark_res.rest.faulty:
+                rest_df.loc[len(rest_df)] = {'file_id': get_file_id(filepath), 'duration': benchmark_res.rest.duration, 'wer': None}
+            else:
+                if (res := eval_rest(benchmark_res.rest, filepath, benchmark_res.expected_transcription)) is None:
+                    rest_df.loc[len(rest_df)] = {'file_id': get_file_id(filepath), 'duration': benchmark_res.rest.duration, 'wer':None}
+                else:
+                    rest_df.loc[len(rest_df)] = asdict(res)
+
+        if benchmark_res.websocket is not None: 
+            if benchmark_res.websocket.faulty:
+                websocket_df.loc[len(websocket_df)] = {'file_id': get_file_id(filepath), 'duration': benchmark_res.rest.duration, 'wer': 'Err', 'average_levenshtein_distance': 'Err'}
+            else:
+                if (res := eval_websocket(benchmark_res.websocket, filepath, benchmark_res.expected_transcription)) is None:
+                    websocket_df.loc[len(websocket_df)] = {'file_id': get_file_id(filepath), 'duration': benchmark_res.rest.duration, 'wer': 'Err', 'average_levenshtein_distance': 'Err'}
+                else:
+                    websocket_df.loc[len(websocket_df)] = asdict(res)
+    if rest_df.size == 0:
+        return websocket_df
+    if websocket_df.size == 0:
+        return websocket_df
+    return pandas.merge(rest_df, websocket_df, on='file_id', how='inner', suffixes=('_rest', '_websocket'))
+
+def eval_rest(rest_benchmark_result: RestResult, result_filepath: str, expected: str) -> RestEvalResult | None:
+    expected_transcription = TRANSFORM_DEFAULT(expected)
+    actual_transcription = TRANSFORM_DEFAULT(rest_benchmark_result.transcript)
+    if len(expected_transcription) == 0 or len(actual_transcription) == 0:
+        return None
+    wer = jiwer.wer(" ".join(actual_transcription), " ".join(expected_transcription))
```

Comment:
can't jiwer.wer handle lists directly??

```suggestion
    wer = jiwer.wer(actual_transcription, expected_transcription)
```